### DISCUSSION TITLE
overlay the path on floor plan

### DISCUSSION THREAD

**Discussion by matroskin-the-cat** [stella-cv/stella_vslam#277] [Q&A] [answered]
I want to overlay the path from the map file on a floor plan and I noticed that no matter how I scale the path it doesn't match the floor plan in some places. Some parts have drifted or the scaling has changed. Can the graph be corrected  If I know that for example frame 1 is at  floor plan pixel coordinates 202, 403 and frame 513 should be at 937,101 and frame 819 at 417, 978?

**Comment by ymd-stella** [comment, marked as answer]
See https://github.com/stella-cv/stella_vslam/blob/139ff815c2c8a1af084417e1d4f48ae2eca14780/src/stella_vslam/optimize/graph_optimizer.cc.

**Reply by matroskin-the-cat** [reply]
I am not familiar with g2o or graph optimization in general but I assume I need  a constrain edge with my data, correct??

**Reply by ymd-stella** [reply]
Yes.
